astro/src/content/blog/fusionauth-and-terraform.mdx

```diff
@@ -0,0 +1,64 @@
+---
+publish_date: 2025-05-15
+title: "Why Use Terraform With FusionAuth"
+description: 'Learn why using Terraform with FusionAuth is beneficial.'
+authors: Mark Robustelli
+image: /img/blogs/fusionauth-and-terraform/fusionauth-and-terraform.png
+categories: Tutorial
+tags: deployment, terraform, fusionauth, deployment
+excerpt_separator: "{/* more */}"
+---
+
+import Aside from '/src/components/Aside.astro';
+
+Let's be honest — the only thing better than a fully working auth system is never having to manually configure it again. Enter FusionAuth and Terraform.
+
+In this post, you will learn why you would want to use the Terraform provider for FusionAuth. You can ditch the manual clicks and start defining your identity resources as code.
+
+<Aside type="tip">TLDR; I just want the guide. You're in luck. FusionAuth just released v1.0.0.0 of the Terraform provider for FusionAuth. You can find the guide on how to use it [here](/docs/operate/deploy/terraform).</Aside>
+
+## Wait... What's Terraform?
+Terraform is a powerful open-source Infrastructure as Code (IaC) tool from HashiCorp. It lets you manage and provision infrastructure and services through human-readable configuration files. Think of it as writing down what your infrastructure should look like and letting Terraform handle the how.
```

Comment:
```suggestion
Terraform is a powerful Infrastructure as Code (IaC) tool from HashiCorp. (OpenTofu is an open-source fork with comparable functionality.) IaC tools let you manage and provision infrastructure and services like databases and servers through human-readable configuration files. They are declarative, which means you write down what your infrastructure should look like and let the tool handle the how.
```

Reply:
I don't know if mentioning OpenTofu is clutter, but Terraform is no longer open source: https://github.com/hashicorp/terraform/commit/b145fbcaadf0fa7d0e7040eac641d9aef2a26433

Reply:
You could just mention OpenTofu here and then say that everything you wrote applies just as much to OpenTofu as it does to Terraform, but you're going to refer to Terraform for the rest of the post.